File: Python/01-Arrays/Ejercicio4.py
# Escribe una función iterativa que dado un número entero n, retorne un array con todos los números
# enteros en orden decreciente desde n a 1.

numeros=[]
numero=int(input("Ingrese un numero: "))

for i in range(1, numero+1):
  numeros.append(numero)
  numero-=1

numeros.sort(reverse=True)
print(numeros)

File: JavaScript/04-Iteradores/Ejercicios/Ejercicio4.js

// 4.	Pedir 15 números y escribir la suma total.

let i=0;
let suma=0;
let n=0;

while (i <= 15) {
    n=parseInt(prompt("Ingrese un numero: "));
    suma+=n;
    i++
}

console.log(`La suma de los 15 numeros ingresador es: ${suma}`)